.github/workflows/build-images.yml

```diff
@@ -27,20 +27,27 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2.4.0
 
-      - name: Validate docker images
-        if: ${{ github.event_name == 'pull_request' }}
-        run: docker build services/${{ matrix.service }}/ --build-arg PHP_VERSION=${{ matrix.php }}
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
 
-      - name: Build and push Docker images
-        uses: docker/build-push-action@v1
-        if: ${{ github.event_name != 'pull_request' }}
+      - name: Login to DockerHub
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1
```

Comment:
This is relevant to current work I am doing, so thank you!

Reply:
🎉